### DISCUSSION TITLE
Question about Sequences

### DISCUSSION THREAD

**Discussion by JonasMueller1991** [sngular/kloadgen#164] [Q&A] [answered]
Hey again,
thanks for the fast answers last time and the quick fix of our problem :)!
With the latest merge concerning the sequence function we run in some issues. Before the merge, it was possible for us to create a sequence by using `[{3,2,1]` in field values list, where the field type is a `long`. This input produced the sequence of `3->2->1`, while now we get a casting error (we also used `seq` as field type). If we drop the bracket '{' we get a sequence with ascending numbers `3->4->5`. 
Comparing it to the string case e.g. `[{a,b,c]` still produce `a->b->c'. Is this intentional that we have different behaviours if we provide an input for field values with the types string and long? 

We wanted to use sequence to give a specific order for `long`-types field for instance `[5,4,3,2,1]` to get subsequently `5->4->3->2->1`, is this still possible?

Furthermore, we detected some issues regarding `bytes_decimal`-typed fields in sequences. Would it be possible to use this type for a sequence?

Thanks in advance and cheers,
Jonas

**Comment by Rivarsal** [comment, marked as answer]
Hello! We found out there is a small issue with the way we changed the code for sequences. We will work on it today, and we will try to have a solution soon. Sorry for the inconveniences.

**Reply by JonasMueller1991** [reply]
Thanks! :)
